### ISSUE TITLE
Archive repo

### ISSUE THREAD

**Issue by tristan-f-r** [gluesql/gluesql-js#3]
This repository is no longer used.

**Comment by ever0de** [comment]
This has been moved to the gluesql main repo.
https://github.com/gluesql/gluesql/tree/main/pkg/javascript

**Comment by LeoDog896** [comment]
Yes, which gives the repository good justification to be archived.

**Comment by panarch** [comment]
revived
